app/presenters/layouts/footer_presenter.rb

```diff
@@ -7,10 +7,11 @@ def student?
     def nav_links
       footer_links = current_user.present? ? [{ title: 'Home', url: '/' }, { title: 'Dashboard', url: '/dashboard' }] : []
 
+      target_blank = { target_blank: { target: '_blank', rel: 'noopener' } }
       custom_links = SchoolLink.where(
         school: current_school,
         kind: SchoolLink::KIND_FOOTER
-      ).map { |sl| { title: sl.title, url: sl.url } }
+      ).map { |sl| { title: sl.title, url: sl.url }.merge(target_blank) }
```

Comment:
why not just 
```ruby
      custom_links = SchoolLink.where(
         school: current_school,
         kind: SchoolLink::KIND_FOOTER
       ).map do |sl|
         { title: sl.title, url: sl.url, attributes: { target: '_blank', rel: 'noopener' }  }
       end
```

Reply:
Let's decide on the attributes to inject in the view, and just pass along the _information_ we intend to use, instead.

```ruby
custom_links = SchoolLink.where(
  school: current_school,	        school: current_school,
  kind: SchoolLink::KIND_FOOTER	        kind: SchoolLink::KIND_FOOTER
).map { |sl| { title: sl.title, url: sl.url, custom: true } }
```

Reply:
This way separated the responsibility of presenter and view precisely.
I learned an important lesson from this.